API.md

```diff
@@ -48,6 +48,61 @@ curl https://host/projects
 ]
 ```
 
+### get-project-tag
```

Comment:
get we rename this to get-project-tags, since it returns a list of tags?

Reply:
👍 Will do!